project1/src/test/java/uk/co/cpsd/javaproject1/GoatTest.java

```diff
@@ -0,0 +1,97 @@
+package uk.co.cpsd.javaproject1;
```

Comment:
I'm just going to admit I haven't looked at the test files because I'm really tired and it's almost 2300 :)
Sorry.